docs/modules/ROOT/pages/references/composite/dbaas/exoscale/postgres.adoc

```diff
@@ -0,0 +1,11 @@
+= pkg.appcat.composite.dbaas.exoscale.postgres
+
+The parent key for all of the following parameters is`pkg.appcat.composite.dbaas.exoscale.postgres`.
+
+== `defaultCompositionRef`
+
+[horizontal]
+type:: string
+default:: `${facts:cloud}.exoscalepostgresqls.appcat.vshn.io`
+
+The default composition to use when a user creates a Claim of the composite resource.
```

Comment:
As mentioned earlier, there's no need for multiple compositions, this section can be removed
```suggestion
```